.azurepipelines/GetCargoMake.yml

```diff
@@ -0,0 +1,114 @@
+## @file
+# Azure Pipeline to download Cargo Make and save it as a pipeline artifact that
+# can be accessed by other pipelines.
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+schedules:
+# At 1:00 on Monday
+# https://crontab.guru/#0_1_*_*_1
+- cron: 0 1 * * 1
+  branches:
+    include:
+    - main
+  always: true
+
+jobs:
+- job: Update_Cargo_Make
+  displayName: Update Cargo Make
+
+  pool:
+    vmImage: windows-latest
+
+  steps:
+  - checkout: none
+    clean: true
+    fetchTags: false
+    persistCredentials: True
+
+  - script: pip install requests --upgrade
+    displayName: Install and Upgrade pip Modules
+    condition: succeeded()
+
+  - task: PythonScript@0

```

Comment:
Any thoughts to making a scripts folder and just provide the python script instead of having it inline? I don't know about other scenarios, but at least stops you from having so much duplicated code between getcargomake and getcargotarpaulin

Reply:
They didn't start out the same but morphed into roughly the same steps toward the end. I was considering a YAML file that wrapped an internal PythonScript step where the template parameters feed into the script. I think a separate script is a more portable and I'll look into adding that.

Reply:
Moved to a separate Python script file called from the YAML files.